src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json

```diff
@@ -4,21 +4,23 @@
             "name": "tab1",
             "title": "YOUR CONTENT",
             "subtitle": "Publish boldly with WordPress Blocks.",
-            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate layout assembly and storytelling with premade designs.",
+            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate telling your story or tending your store with professional designs.",
```

Comment:
Is the second **store** use intentional? Or should that be switched with **site** or **website**?

Reply:
This one was intentional to exclude from replacements, but thanks for checking!